packages/cmf/__tests__/selectors/components.test.js

```diff
@@ -0,0 +1,62 @@
+import { fromJS } from 'immutable';
+import selectors from '../../src/selectors';
+
+describe('selectors.components', () => {
+	const formData = {
+		data: { type: 'engine' },
+		dirty: false,
+	};
+	const state = {
+		cmf: {
+			components: fromJS({
+				'Container(Form)': {
+					MY_FORM_ID: formData,
+				},
+				'Container(List)': {
+					items: [],
+				},
+			}),
+		},
+	};
+	describe('toJS', () => {
```

Comment:
`describe` execute a test ?
Same for next tests.

I don't know if that executes well, but try to follow the purpose of jest functions
* `describe` : group tests under a descriptive label. Ex: `describe('toJS')`
* `it`: unit test, try to make it english readable. Ex: `it('should do that when we do this')`

In this case `it('should get requested state and memoize it')`

Why ?
Here the test report will output `selectors.components toJS` which is not a good description of the test.
With the previous suggestion, the test report will output `selector.components toJS should do that when we do this` --> en readable.